test/fulfillment_order_test.rb

```diff
@@ -49,5 +48,80 @@ def setup
         assert_equal 450789469, fulfillment.order_id
       end
     end
+
+    context "#move" do
+      should "be able to move fulfillment order to a new_location_id" do
+        fulfillment_order = ShopifyAPI::FulfillmentOrder.find(519788021)
+        new_location_id = 5
+
+        moved = ActiveSupport::JSON.decode(load_fixture('fulfillment_order'))
+        moved['assigned_location_id'] = new_location_id
+        fulfillment_order.status = 'closed'
+
+        body = {
+            original_fulfillment_order: fulfillment_order,
+            moved_fulfillment_order: moved,
+            remaining_fulfillment_order: nil,
+        }
+        api_version = ShopifyAPI::ApiVersion.find_version('2019-01')
+        endpoint = "fulfillment_orders/519788021/move"
+        extension = ".json"
+        url = "https://this-is-my-test-shop.myshopify.com#{api_version.construct_api_path("#{endpoint}#{extension}")}"
+        url = url + "?fulfillment_order%5Bnew_location_id%5D=5"
+        fake endpoint, :method => :post, :url => url, :body => ActiveSupport::JSON.encode(body)
```

Comment:
what does the `url` argument do?

Reply:
so the 'endpoint' is the short path form, and 'url' is the full url long form. As I mentioned there's a simpler way to achieve what's needed here.